### PR TITLE
Update Greek (el) translation

### PR DESCRIPTION
**Pull request by dglent**
Update of the Greek (el) translation, add the following files:
        lxqt-leave/translations/lxqt-hibernate_el_GR.desktop
        lxqt-leave/translations/lxqt-leave_el_GR.desktop
        lxqt-leave/translations/lxqt-leave_el_GR.ts
        lxqt-leave/translations/lxqt-lockscreen_el_GR.desktop
        lxqt-leave/translations/lxqt-logout_el_GR.desktop
        lxqt-leave/translations/lxqt-reboot_el_GR.desktop
        lxqt-leave/translations/lxqt-shutdown_el_GR.desktop
        lxqt-leave/translations/lxqt-suspend_el_GR.desktop

### CODE REVIEW

File: lxqt-config-session/translations/lxqt-config-session_el_GR.ts [hunk 1]
```diff
@@ -1,35 +1,35 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version='1.0' encoding='utf-8'?>
 <!DOCTYPE TS>
 <TS version="2.1" language="el_GR">
 <context>
     <name>AutoStartEdit</name>
     <message>
-        <location filename="../autostartedit.ui" line="14"/>
+        <location line="14" filename="../autostartedit.ui"/>
```
Comment (palinek) [inline lxqt-config-session/translations/lxqt-config-session_el_GR.ts]: There are many unnecessary changes that logicaly mean noop (e.g. here just attributes reordering). Can you, please, get rid of them?

Comment (dglent) [inline lxqt-config-session/translations/lxqt-config-session_el_GR.ts]: I will try maybe to compile lxqt with the new translation so it will regenerate the file with this syntax (?) 
Normaly it is nt a problem as in next lupdate they will be changed again. 
This is due the Lokalize program which i use for translations because i have a big translations memory/glossary etc. and it helps me a lot to translate fast.

**Comment by palinek** [comment]
landed in lxde/lxqt-session@6834967 (i've modified your commit by regenerating the .ts files with `lupdate`)

thanks
